donate/payments/tests/test_views.py

```diff
@@ -967,3 +969,33 @@ def test_get_initial_populates_email_from_session(self):
         self.assertEqual(view.get_initial(), {
             'email': 'test@example.com',
         })
+
+
+class TestApplePayDomainAssociationView(TestCase):
+    def setUp(self):
+        self.view_url = '/.well-known/apple-developer-merchantid-domain-association'
+        self.factory = RequestFactory()
+
+    def test_view_returns_key(self):
+        """
+        Make sure the view returns the apple pay key when set
+        """
+        settings.APPLE_PAY_DOMAIN_ASSOCIATION_KEY = "test_apple_pay_key"
+        request = self.factory.get(self.view_url)
+
+        response = apple_pay_domain_association_view(request)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content.decode(), "test_apple_pay_key")
```

Comment:
What do you think about using the [`assertContains`](https://docs.djangoproject.com/en/4.1/topics/testing/tools/#django.test.SimpleTestCase.assertContains) method? It can check status and content in one go without the need to `decode` the `content`.

Reply:
Great suggestion Tibor! Updated 👍